test/apps/iodemo/io_demo.cc

```diff
@@ -1410,6 +1409,39 @@ class DemoClient : public P2pDemoCommon {
         return data_size;
     }
 
+    void
+    get_server_info_uncompleted_ops(const server_info_t& server_info,
+                                    std::stringstream &ss) const
+    {
+        ss << get_num_uncompleted(server_info) << " uncompleted operations"
+           << " (read: " << server_info.num_completed[IO_READ] << "/"
+           << server_info.num_sent[IO_READ] << "; write: "
+           << server_info.num_completed[IO_WRITE] << "/"
+           << server_info.num_sent[IO_WRITE] << ")";
+    }
+
+    void get_uncompleted_servers_info(std::stringstream &list_uncomp_ss,
+                                      size_t &uncomp_to_disconnect)
```

Comment:
maybe just return size_t from this func?

Reply:
done